src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -436,7 +436,7 @@ function transitionTracks(
                 heatmap01 = trackSpecKeyToTrackId[spec.key];
             } else if (heatmap === undefined 
                         && spec.molecularAlterationType !==  AlterationTypeConstants.METHYLATION 
-                        && spec.molecularAlterationType !==  AlterationTypeConstants.TREATMENT_RESPONSE) {
+                        && spec.molecularAlterationType !==  AlterationTypeConstants.GENERIC_ASSAY) {
```

Comment:
How come we don't use this enum everywhere?  (like, all the places above where we have "GENERIC_ASSAY" as string?

Reply:
@alisman I noticed this when I change it last time for passing the tests. This was implemented in the `treatment frontend implementation`. I will definitely change those when I implement the of the `generic_assay` frontend as the next step. I think we can leave as it for now, I will change those to use enum later.